### ISSUE TITLE
Overload method C++

### ISSUE THREAD

**Issue by JosenJP** [multilang-depends/depends#2]
It seems overload methods of C++ are not handled yet.

**Comment by gangz** [comment]
Yes, the overload methods of all languages have not been handled yet. Currently, it simply treat all overloaded methods as one. 
should be handled in future (to distinguish the overloaded methods based on parameter list)
